types/jasmine/index.d.ts

```diff
@@ -159,6 +160,18 @@ interface DoneFn extends Function {
     fail: (message?: Error | string) => void;
 }
 
+/**
+ * Configuration that can be used when configuring Jasmine via {@link jasmine.Env.configure}
+ */
+interface EnvConfiguration {
```

Comment:
Sorry for the late notice, overlooked that during previous iteration. Could you please move it under `jasmine` namespace? That helps to not pollute the global scope with jasmine specific types.